source/server/overload_manager_impl.cc

```diff
@@ -106,6 +113,37 @@ Stats::Gauge& makeGauge(Stats::Scope& scope, absl::string_view a, absl::string_v
 
 } // namespace
 
+NamedOverloadActionSymbolTable::Symbol
+NamedOverloadActionSymbolTable::get(absl::string_view string) {
+  if (auto it = table_.find(string); it != table_.end()) {
+    return Symbol(it->second);
+  }
+
+  size_t index = table_.size();
+
+  names_.emplace_back(string);
+  table_.emplace(std::make_pair(string, index));
+
+  return Symbol(index);
+}
+
+absl::optional<NamedOverloadActionSymbolTable::Symbol>
+NamedOverloadActionSymbolTable::find(absl::string_view string) const {
```

Comment:
tiny nit: STL has trained me to expect .find() to return an iterator. Maybe ::get(), which more intuitively gets a nullable wrapper around the thing? That way there's an obvious GetOrDie() parallel (not that you need to implement that).